tests/TwigExtensionTest.php

```diff
@@ -12,11 +12,73 @@
 use Psr\Http\Message\ResponseFactoryInterface;
 use Psr\Http\Message\UriInterface;
 use Slim\Interfaces\CallableResolverInterface;
+use Slim\Interfaces\RouteInterface;
 use Slim\Routing\RouteCollector;
 use Slim\Views\TwigExtension;
 
 class TwigExtensionTest extends TestCase
 {
+    /**
+     * Create a route collector using the given base path.
+     *
+     * Note that this method would create mocks for the ResponseFactoryInterface
+     * and the CallableResolverInterface injected into the constructor of the
+     * RouteCollector.
+     *
+     * @param string $basePath
+     *
+     * @return RouteCollector
+     */
+    protected function createRouteCollector(string $basePath): RouteCollector
+    {
+        $routeCollector = new RouteCollector(
+          $this->createMock(ResponseFactoryInterface::class),
+          $this->createMock(CallableResolverInterface::class)
+        );
+        if ($basePath) {
```

Comment:
Please add a line return here. My OCD is going wild 😂

Reply:
Done.